### DISCUSSION TITLE
NextJS GET request for the _error page denied when using basic auth in Internet Explorer

### DISCUSSION THREAD

**Discussion by jfarris587** [vercel/next.js#10709] [Help]
**All browsers (including Microsoft Edge) work, except for Internet Explorer** when navigating an `_error` page. If a user tries to navigate to a route that is not supported `localhost:3000/banana` then nextJS will call `_error` to render the page. However, in IE, this fails. Console will error the following:

![image](https://user-images.githubusercontent.com/31263423/75376487-21996700-589e-11ea-9203-101edfd9e55c.png)

**The problem**

This only happens when using basic auth. So looking into the Network logs I find that the GET request to `_error` is denied and a 401 response is sent.

![image](https://user-images.githubusercontent.com/31263423/75377743-6f16d380-58a0-11ea-8846-7567c09198e7.png)

However, this only happens for the error page, not for any other page in nextJS. Looking at what happens when loading any other page, this is the result - one GET request that is a 401, and then another immediately after that is successful.

Like the request for `_error` page, the first request contained no Authorization in the header, but the successful request did have it.

![image](https://user-images.githubusercontent.com/31263423/75377808-8f469280-58a0-11ea-9b35-f004a8ffca1c.png)

**Comment by timneutkens** [comment]
Looks strange! Could you create a minimal repository that we can investigate on?
